.github/workflows/realease-on-pr-merge.yaml

```
@@ -39,23 +39,22 @@ jobs:
         run: |
           git config --global user.name "SeungBin Kim"
           git config --global user.email "luxorienskim@gmail.com"
-      - name: Bump version and push tag
+      - name: Bump NPM Version (Without Commit)
         id: bump_version
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:10:28: Double quote to prevent globbing and word splitting

(shellcheck)